onnxruntime/contrib_ops/cpu/bert/attention_cpu_base.h

```diff
@@ -257,12 +286,15 @@ class AttentionCPUBase : public AttentionBase {
                                const T* past_value,       // past value only (if not using past state)
                                T* present,                // present state
                                T* present_value,          // present value only (if not using present state)
-                               ThreadPool* tp) const {
+                               ThreadPool* tp,
+                               bool use_dmmha = false,
```

Comment:
rename use_dmmha to past_present_share_buffer (consider that cross attention of dmmha not set this flag).